### ISSUE TITLE
conda-build 2.1.0 doesn't like prefix in swig 3.0.2-0

### ISSUE THREAD

**Issue by guyer** [conda/conda-build#1619]
I heeded the warning and updated conda-build from 2.0.12 to 2.1.0. Afterwards, my build generates the following warnings:

```
WARNING conda_build.build:create_env(685): Build prefix failed with prefix length 255
WARNING conda_build.build:create_env(686): Error was: 
WARNING conda_build.build:create_env(687): Placeholder of length '80' too short in package swig-3.0.2-0.
The package must be rebuilt with conda-build > 2.0.
WARNING conda_build.build:create_env(688): One or more of your package dependencies needs to be rebuilt with a longer prefix length.
WARNING conda_build.build:create_env(690): Falling back to legacy prefix length of 80 characters.
WARNING conda_build.build:create_env(691): Your package will not install into prefixes > 80 characters.
updating index in: /Users/guyer/anaconda/conda-bld/osx-64
updating index in: /Users/guyer/anaconda/conda-bld/noarch
```

swig-3.0.2 is from the default channel. How do I get that rebuilt so that my package will "install into prefixes > 80 characters"?

**Comment by msarahan** [comment]
I recommend using a newer version of SWIG.  3.0.10 is available, and does not have this issue.

**Comment by guyer** [comment]
I originally built with swig 3.0.11 from conda-forge and the build product didn't work because of some abstract base class that swig didn't properly deal with. I rolled back to 3.0.2 to see if that helped (it did), but then I discovered this prefix issue when I tried rebuilding (for unrelated reasons).

This library (Trilinos) takes hours to build and I'm not really keen on trying every minor version of swig (why do they insist on breaking everything on every dot-dot release?!?).

I'm having a hard time deciding whether I hate Trilinos or SWIG more.
